src/components/offer/offer.tsx

```diff
@@ -0,0 +1,95 @@
+import { offerTypes } from '../../const';
+import { type Offer, OfferFull } from '../../types/offer';
+import { Review } from '../../types/review';
+import Bookmark from '../bookmark/bookmark';
+import Map from '../map/map';
+import ReviewsList from '../reviews-list/reviews-list';
+import StarsRating from '../stars-rating/stars-rating';
+
+type OfferProps = {
+  offer: OfferFull;
+  nearOffers: Offer[];
+  reviews: Review[] | null;
```

Comment:
null здесь не нужен, если массив отзывов будет пустой ты просто их не будешь отображать
```suggestion
  reviews: Review[];
```